Application/Telegram/TelegramCommunicationService.cs

```diff
@@ -28,23 +24,59 @@ public TelegramCommunicationService(IConfiguration configuration, IMediator medi
 
         public async Task Execute(Update update)
         {
+            await _telegraBotClient.SendChatActionAsync(update.Message.Chat.Id, ChatAction.Typing);
             if (update.Message != null)
             {
                 switch (update.Message.Text)
                 {
                     case "/start":
                         await _telegraBotClient.SendTextMessageAsync(update.Message.Chat.Id, "To get all workplaces use command /getworkplaces");
-                        return;
+                        break;
                     case "/getworkplaces":
                         var result = await _mediator.Send(new GetWorkplaceListQueryRequest());
                         foreach (var item in result.Results)
                         {
                             await _telegraBotClient.SendTextMessageAsync(update.Message.Chat.Id, $"WorkplaceId:{item.Id},WorkplaceNumber:{item.WorkplaceNumber},");
                         }
-                        return;
+                        break;
+                    case "/bookings":
+                        
+                        var bookingResponse = await _mediator.Send(new GetBookingListQueryRequest());
```

Comment:
you can move this into SendBookingList() so here it will be just call this method and that's it